### ISSUE TITLE
Crash when using configurable new tab

### ISSUE THREAD

**Issue by ttttmr** [felixse/FluentTerminal#351]
Version: FluentTerminal.App_0.4.1.0_Test in Windows 10 1809

I created a few profiles, and when I use Ctrl+Shift+T, it automatically quits.

I tried it many times. I found that it will crash when I use Ctrl+Shift+T after creating a new profile, but when I delete one of them, Ctrl+Shift+T will work. This is too strange.

I found some CrashDumps files(like FluentTerminal.App.exe.xxx.dmp), I can upload them to you if you need them.

**Comment by PadreSVK** [comment]
I have similiar issue, could you send please terminal profiles?

**Comment by ttttmr** [comment]
I added some ssh (like `ssh user@host`), and VS command line for x84 and x64( like `%comspec% /k "VisualStudioPATH\VC\Auxiliary\Build\vcvars64.bat"`)

**Comment by felixse** [comment]
Can you enable [debug logging](https://github.com/felixse/FluentTerminal/wiki/Logging) and post some logs?

**Comment by ttttmr** [comment]
I just reproduced this question, this is some logs, it crashed when I pressed Ctrl+Shift+T.

```
2019-05-07 09:12:38.734 +08:00 [INF] Initialized
2019-05-07 09:12:38.765 +08:00 [DBG] Created ApplicationViewAdapter for ApplicationView with Id: [-919037]
2019-05-07 09:12:42.938 +08:00 [DBG] WebView navigation completed. Target: ["ms-appx-web://53621fsapps.fluentterminal/Client/index.html"]
2019-05-07 09:12:43.713 +08:00 [DBG] Received GetAvailablePortResponse: [{"Port":49151,"$type":"GetAvailablePortResponse"}]
2019-05-07 09:12:43.713 +08:00 [INF] Server started at ws://127.0.0.1:49151 (actual port 49151)
2019-05-07 09:12:43.713 +08:00 [DBG] WebSocketServer started. Calling connectToWebSocket() now.
2019-05-07 09:12:43.775 +08:00 [DBG] WebSocket open
2019-05-07 09:12:43.790 +08:00 [DBG] Sending CreateTerminalRequest: [{"Id":0,"Size":{"Columns":114,"Rows":36,"$type":"TerminalSize"},"Profile":{"Id":"813f2298-210a-481a-bdbf-c17bc637a3e2","PreInstalled":true,"Name":"Powershell","Arguments":"","Location":"C:\\windows\\system32\\WindowsPowerShell\\v1.0\\powershell.exe","WorkingDirectory":"","TabThemeId":0,"LineEndingTranslation":"DoNotModify","TerminalThemeId":"00000000-0000-0000-0000-000000000000","KeyBindings":[{"Command":"813f2298-210a-481a-bdbf-c17bc637a3e2","Key":49,"Ctrl":true,"Alt":true,"Shift":false,"Meta":false,"$type":"KeyBinding"}],"$type":"ShellProfile"},"SessionType":"WinPty","$type":"CreateTerminalRequest"}]
2019-05-07 09:12:44.578 +08:00 [DBG] Received CreateTerminalResponse: [{"Success":true,"Error":null,"ShellExecutableName":"powershell","$type":"CreateTerminalResponse"}]
2019-05-07 09:12:54.273 +08:00 [DBG] Received keyboard command: '["ConfigurableNewTab"]'
2019-05-07 09:12:54.397 +08:00 [ERR] Unhandled Exception
System.NullReferenceException: Object reference not set to an instance of an object.
   at FluentTerminal.Models.ShellProfile.Equals(Object) + 0x9f
   at System.Collections.Generic.ObjectEqualityComparer`1.Equals(T, T) + 0x1d
   at SharedLibrary!<BaseAddress>+0x8035b9
   at System.Array.IndexOf[T](T[], T, Int32, Int32) + 0x52
   at System.Collections.Generic.List`1.IndexOf(T) + 0x43
   at System.Collections.Generic.List`1.System.Collections.IList.IndexOf(Object) + 0x5e
   at System.Runtime.InteropServices.WindowsRuntime.IBindableVector_CCWAdapter.IndexOf(IList, Object, UInt32&) + 0x14
   at System.Collections.IList__Impl.Vtbl.IndexOf__n(IntPtr, Void*, UInt32*, SByte*) + 0x77
   at System.Runtime.InteropServices.McgMarshal.ThrowOnExternalCallFailed(Int32, RuntimeTypeHandle) + 0x21
   at __Interop.ComCallHelpers.Call(__ComObject, RuntimeTypeHandle, Int32, Void*) + 0xbe
   at __Interop.ForwardComStubs.Stub_70[TThis](__ComObject, Object, Int32) + 0x48
   at FluentTerminal.App.Dialogs.ShellProfileSelectionDialog.ShellProfileSelectionDialog_obj1_Bindings.Update() + 0x4b
   at FluentTerminal.App.Dialogs.ShellProfileSelectionDialog.ShellProfileSelectionDialog_obj1_Bindings.Loading(FrameworkElement, Object) + 0xe
   at System.Xml.XmlTextReaderImpl.OnDefaultAttributeUseDelegate.Invoke(IDtdDefaultAttributeInfo, XmlTextReaderImpl) + 0x2e
   at __Interop.Intrinsics.HasThisCall__22[TArg0](Object, IntPtr, Object, TArg0) + 0x36
   at FluentTerminal.App!<BaseAddress>+0x111b22e
```

**Comment by felixse** [comment]
should be fixed by 4ec9f0a5
